framework/src/st_things/things_stack/things_stack.c

```diff
@@ -175,7 +175,7 @@ int things_initialize_stack(const char *json_path, bool *easysetup_completed)
 		abs_json_path = things_make_abs_device_def_path(json_path);
 		THINGS_LOG_V(TAG, "Origin path(%s), converted path(%s).", json_path, abs_json_path);
 	}
-#if CONFIG_ST_THINGS_FOTA
+#ifdef CONFIG_ST_THINGS_FOTA
```

Comment:
some files using `#if define(xxx)`
but here is using "#ifdef"
in my opinion it would be better to unify into one.

Reply:
ok.